src/components/admin/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { Fragment } from "react";
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Dialog, Transition } from "@headlessui/react";
+
+import { SidebarContent } from "./components/SidebarContent";
+
+export interface SidebarProps {
+  sidebarOpen: boolean;
+  onCloseSidebar: () => void;
+}
+
+export const Sidebar: React.FC<SidebarProps> = ({
+  onCloseSidebar,
+  sidebarOpen,
```

Comment:
[OOS] Might be a good idea to define a common interface for modals and popups uniforming the same `open` or `show` or `visible` and `on*` props for similar components